### PR TITLE
simple table with mock data

### PR DESCRIPTION
**Pull request by Mel-dev-hub**
Created simple table with mock data from backend.

### CODE REVIEW

File: frontend/src/components/GeocacheList/GeocacheList.tsx [hunk 1]
```diff
@@ -1,7 +1,93 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { Geocache } from "../../model/Geocache";
+
 const GeocacheList = () => {
+    const [notes, setNotes] = useState([]);
```
Comment (kemm0) [inline frontend/src/components/GeocacheList/GeocacheList.tsx:6]: Maybe the variable name would be more descriptive if it was geocaches instead of notes

**Review by kemm0 — CHANGES_REQUESTED** [review]
I added some notes that should be fixed before merging

**Review by Mel-dev-hub — COMMENTED** [review]
useEffect function is a bit different than suggested, changed api method a little. Let me know what you think.

**Comment by kemm0** [comment]
I think this is good for the first version 👍

**Review by kemm0 — APPROVED** [review]

**Pull request merged**